### ISSUE TITLE
Name and save data stores int ~/.arcana/stores.yml using `arcana store add`

### ISSUE THREAD

**Issue by tclose** [Australian-Imaging-Service/arcana#12]
Create a new CL tool, `arcana store add` that saves connection information (e.g. server, user-alias, user-secret) for XNAT repositories/stores in a per-user configuration file (e.g. ~/.arcana/stores.yml) to save having to re-enter it each time a user is are working with a particular store. Something simple like this should work I think

```yaml
- usyd_xnat:
    type: arcana.data.stores.medicalimaging:Xnat
    server: https://xnat.sydney.edu.au
    alias: LWDJW-SWWD-NCEFE-FWEF
    secret: qwe;fwh;fo2i321rwef;2k;ljf23l;krj
- uq_xnat:
    type: arcana.data.stores.medicalimaging:Xnat
    server: https://xnat.uq.edu.au
    alias: QYITE-FSEED-MHXSF-OUWE
    secret: 3k3p097#@#HWFEL(f9329#9s-!
```

The desired behaviour can be viewed at https://arcana.readthedocs.io/en/develop/data_model.html#stores. The API should use `DataStore.save()` and `DataStore.load()` under the hood to save and load the YAML configurations

**Comment by tclose** [comment]
@ablachair I realised when I went to debug the unittests on the develop branch that I had already implemented a lot of this issue, so I have finished it off and pushed the changes to develop. I have also added a unittest for it at arcana/cli/tests/test_store_cli.py, which might be useful as a reference.

Really sorry if you have already started work on this, but I thought it would be easier if I just finished off what I had started instead of getting you to come in halfway through.
